ChangeLog

```diff
@@ -61,6 +61,10 @@ Features
    * Add MBEDTLS_REMOVE_3DES_CIPHERSUITES to allow removing 3DES ciphersuites
      from the default list (enabled by default). See
      https://sweet32.info/SWEET32_CCS16.pdf.
+   * Add support for parsing subject alternative name, of type otherName,
```

Comment:
Minor: RFC should be capitalized.

Reply:
Minor: Improve wording of this entry. Suggestion: `Add support for parsing otherName entries in the Subject Alternative Name X.509 certificate extension. `